Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocListControlObject.cs

```diff
@@ -235,7 +235,7 @@ public BocListCellControlObject GetCellWhere (string columnItemID, string cellTe
       if (column.HasDiagnosticMetadata)
       {
         var cssSelector = string.Format(
-            ".bocListTable .bocListTableBody .bocListDataRow .bocListDataCell[{0}='{1}'] span[{2}={3}]",
+            ".bocListTable .bocListTableBody .bocListDataRow .bocListDataCell[{0}='{1}'] *[{2}={3}]",
```

Comment:
```suggestion
            ".bocListTable .bocListTableBody .bocListDataRow .bocListDataCell[{0}='{1}'] *:is(span, div)[{2}={3}]",
```
Should we try this?

Reply:
We can but I think it does not add any value. We already filter the element for a `boclist-cell-item-content` attribute so no other element can be selected. Having a less specific selector here allows us to change things in the future without having to change/forget them here.

Reply:
I am going to keep the `*`, since we also filter for the `boc-list-cell-content` attribute it should be easy enough to find the relevant source code/HTML